src/test/java/LoginTests.java

```diff
@@ -0,0 +1,83 @@
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class LoginTests extends BaseTest {
+    @Test
+    public void LoginSuccessTest()  {
+
+        openLoginPage();
+        enterEmail("europlatinum@mail.ru");
+        enterPassword("Newjob2!");
+        clickSubmit();
+        String actualText = getSuccessText (".alert.alert-danger");
+        Assert.assertEquals(actualText, "Success");
+    }
+    private String getSuccessText(String s){
+        String st = "Success";
+        return st;
+    }
+    @Test
+    public void LoginWrongCredentialTest() {
+
+         openLoginPage();
+        enterEmail("europlatinum@mail.ru");
+        enterPassword("Newjob2!");
+        clickSubmit();
+        String actualText = getErrorText(".alert.alert-danger");
+        Assert.assertEquals(actualText, "Username or password is incorrect");
+    }
+    @Test
+    public void LoginEmptyEmailTest(){
+        openLoginPage();
+        enterPassword("Newjob2!");
+         enterEmail("");
+        clickSubmit();
+        String actualText = getErrorText("[class='form-group has-error'] div");
+        Assert.assertEquals(actualText, "Email is required");
+   }
+   @Test
+    public void LoginEmptyPasswordTest() {
+       enterPassword("");
+       enterEmail("europlatinum@mail.ru");
+        openLoginPage();
+        clickSubmit();
+        String actualText = getErrorText("[class='form-group has-error'] div");
```

Comment:
you use the same locators in two different tests. You can store this locator in separate variable and then use one locator for all tests